### DISCUSSION TITLE
How to handle multiple inputs in flax, is there any example as referece?

### DISCUSSION THREAD

**Discussion by long1216** [google/flax#1761] [Q&A] [answered]
![keras_multi_input_arch](https://user-images.githubusercontent.com/19679343/148273523-70ee46fe-1c19-4d52-8b31-e2ec3ce3fe43.png)
Is there any way to handle two inputs in flax(like the keras example showing above). Would there be any example showing how to do this? Thank you so much.

**Comment by marcvanzee** [comment, marked as answer]
The seq2seq model has two inputs: https://github.com/google/flax/blob/main/examples/seq2seq/train.py#L262

You simple have two arguments to the `__call__` function of your Module.

Then when you want to run the forward pass you simply provide both arguments to the `apply` function of the Module.
